packages/typespec-ts/src/modular/helpers/operationHelpers.ts

```diff
@@ -645,14 +649,16 @@ function buildBodyParameter(
     bodyParameter.type.type === "byte-array" &&
     !bodyParameter.isBinaryPayload
   ) {
-    addImportToSpecifier("coreUtil", runtimeImports, "uint8ArrayToString");
+    const uint8ArrayToStringReference = resolveReference(
+      AzureCoreDependencies.uint8ArrayToString
```

Comment:
you should be able to do this

```ts
const dependencies = useDependencies();
resolveReference(dependencies.uint8ArrayToString)
```

Which will resolve the appropriate dependency. @azure/core-util if flavor is azure or @typespec/ts-http-runtime otherwise

Reply:
it works, thanks